### PR TITLE
Apply patch

### PR DESCRIPTION
**Pull request by benrudolph**
@snopoke @czue this should do the trick. got it working on staging

cc: @emord

### CODE REVIEW

File: fab/operations/release.py [hunk 1]
```diff
@@ -281,3 +281,16 @@ def mark_keep_until(keep_days):
     until_date = (datetime.utcnow() + timedelta(days=keep_days)).strftime(DATE_FMT)
     with cd(env.code_root):
         sudo('touch {}{}'.format(KEEP_UNTIL_PREFIX, until_date))
+
+
+@roles(ROLES_ALL_SRC)
+@parallel
+def apply_patch(filepath):
+    destination = '/home/{}/patch.patch'.format(env.user)
```
Comment (czue) [inline fab/operations/release.py]: would it make sense to use unique names for these?

**Review by czue — APPROVED** [review]
nice glad this was this simple

File: fab/operations/release.py (hunk 1 above)
Comment (benrudolph) [reply fab/operations/release.py]: could do, my initial logic was i didn't want a bunch of files hanging around, but now thinking about it again it'd be useful to have the date in the filename so we know when it happened

Comment (benrudolph) [reply fab/operations/release.py]: https://github.com/dimagi/commcare-hq-deploy/pull/152/commits/45023919d99808cd6067bc3318a81c6a8ac0cf45

Comment (czue) [reply fab/operations/release.py]: yeah i was thinking that if we did this multiple times we might want a record on the servers of what had happened. and there would be less likelihood of bugs related to applying the wrong patch (not sure how that would happen but was trying to be paranoid)

**Comment by snopoke** [comment]
That was fast.

**Comment by czue** [comment]
@benrudolph is a :1234: :racehorse:

**Review by snopoke — APPROVED** [review]

**Comment by benrudolph** [comment]
:robot:

**Pull request merged**